tests/Feature/Store/VoucherManagerTest.php

```diff
@@ -90,6 +89,75 @@ public function testThreeColumnsAreVisible()
             ->seeElement("#allocate-vouchers");
     }
 
+    /** @test */
+    public function itCanShowFamilyWarnings()
+    {
+        /**
+         * Create
+         * - a registration
+         * - on a center
+         * - in an area that checks for ID
+         * - for a family
+         * - with kids who havn't been checked for ID.
+         */
+
+        // Make a sponsor
+        $sponsor = factory(Sponsor::class)->create();
+
+        // Set the verification and extended rules to match it.
+        Config::set('arc.verifies_children', [$sponsor->shortcode]);
+        Config::set('arc.extended_sponsors', [$sponsor->shortcode]);
+
+        // Make a Centre in it.
+        $centre =  factory(Centre::class)->create([
+            'sponsor_id' => $sponsor->id,
+        ]);
+
+        // Make a family, with unverified kids.
+        $family = factory(Family::class)->create();
+        $unverifiedKids = factory(Child::class, 3)->states('unverified')->make();
+        $family->children()->saveMany($unverifiedKids);
+
+        // Link it all up to a registration.
+        $registration = factory(Registration::class)->create([
+            'centre_id' => $centre->id,
+            'family_id' => $family->id,
+        ]);
+
+        // Make a CentreUser
+        $ccUser = factory(CentreUser::class)->create([
+            "name" => "CC test user",
+            "email" => "testccuser@example.com",
+            "password" => bcrypt('test_ccuser_pass'),
+        ]);
+        $ccUser->centres()->attach($centre->id, [
+            'homeCentre' => true
+        ]);
+
+        // Navigate to the page as that user.
+        $this->actingAs($ccUser, 'store')
+            ->visit(URL::route('store.registration.voucher-manager', [ 'id' => $registration ]))
+        ;
+
+        // We can see the warnings area
+        $this->seeElement("#family-warning");
```

Comment:
this might be `reminder` if changed in code

Reply:
Could probably be "family-notice", will change in next card